src/app.ts

```diff
@@ -16,15 +17,19 @@ import sgMail from "@sendgrid/mail";
 
 import { getAdmin } from "./firebaseAdmin";
 import { verifyFirebaseIdToken } from "./verifyFirebaseIdToken";
+import BadRequestError from "./errors/BadRequestError";
 
 // tslint:disable-next-line:no-var-requires
 const config = require("./config/config.json");
 import {
   coconutApiKey,
   sendgridApiKey
 } from "./config/DO_NOT_COMMIT.secrets.config";
+import { DocumentSnapshot, Firestore } from "@google-cloud/firestore";
 
 const TEN_MINUTES = 1000 * 60 * 10;
+const DEFAULT_DONATION_RECIPIENT = "RAHA";
```

Comment:
Is this supposed to be a user id? If so might as well give it that type for clarity